Remove duplicates from list

sanjaypradeep opened the pull request:
Added one more feasible solution. Complexity is very less.

Commit:
Testing

Testing

Commit:
Given one more effecient solution

hmangukia commented:
1) The code has syntax error, please solve it.
2) Name 'inputWhichHasDuplicates' is not defined
3) Just give a single result removing the duplicate values, no two.

sanjaypradeep committed:
Addresses the comments. 

1. Syntax error is solved.
2. 'inputHasDuplicates' is defined now
3. Removed print statement.
